post_install.sh

```diff
@@ -18,10 +18,54 @@ sed -i '' -e 's/;listen.group = www/listen.group = www/g' /usr/local/etc/php-fpm
 sed -i '' -e 's/;listen.mode = 0660/listen.mode = 0600/g' /usr/local/etc/php-fpm.d/www.conf
 sed -i '' -e 's?;cgi.fix_pathinfo=1?cgi.fix_pathinfo=0?g' /usr/local/etc/php.ini
 
+
+# Write ginx configuration
+cat > /usr/local/etc/nginx/nginx.conf <<EOL
+worker_processes  1;
+
+events {
+    worker_connections  1024;
+}
+
+http {
+    include       mime.types;
+    default_type  application/octet-stream;
+    sendfile        on;
+    keepalive_timeout  65;
+    root /usr/local/bookstack/public;
+
+    server {
+        listen       80;
+        server_name _;
+        index index.php;
+        client_max_body_size 100M;
+
+        location / {
+            try_files $uri $uri/ /index.php?$query_string;
+        }
+
+        location ~ \.php$ {
+          include fastcgi_params;
+          fastcgi_pass            unix:/var/run/php-fpm.sock;
+          fastcgi_index            index.php;
+          fastcgi_buffers            8 16k;
+          fastcgi_buffer_size        32k;
+          fastcgi_param DOCUMENT_ROOT    \$realpath_root;
+          fastcgi_param SCRIPT_FILENAME    \$realpath_root\$fastcgi_script_name;
+        }
+
+        error_page   500 502 503 504  /50x.html;
+            location = /50x.html {
+            root   /usr/local/www/nginx-dist;
+        }
+    }
+}
+EOL
+
```

Comment:
I think this part is not needed, the iocage creation will copy files from the overlay folder into the proper location (see https://github.com/Adirael/iocage-plugin-bookstack/blob/master/overlay/usr/local/etc/nginx/nginx.conf)

Reply:
Yes, good point. Overlooked that, I am pretty new to iocages. I made additional fixes, such as adapting the packages needed in the iocage description.